libs/etterlatte-vilkaarsvurdering-model/src/main/kotlin/vilkaarsvurdering/Vilkaarsvurdering.kt

```diff
@@ -9,7 +9,8 @@ data class VilkaarsvurderingDto(
     val behandlingId: UUID,
     val vilkaar: List<Vilkaar>,
     val virkningstidspunkt: YearMonth,
-    val resultat: VilkaarsvurderingResultat? = null
+    val resultat: VilkaarsvurderingResultat? = null,
+    val yrkesskade: Boolean = false
```

Comment:
Det føles litt malplassert ut som et eget felt på toppnivå 🤔 

Hvis det ikke skal benyttes frontend holder det kanskje å ha det som en metode?

Reply:
Dvs - vilkaarvurdering "eier" det - men det eksekveres i trygdetid?

Reply:
Tanken er vel at trygdetidsappen skal kalle vilkårsvurdering for en behandling og sjekke om det er yrkesskade-
Jeg er enig med deg, den burde ikke være her.
Hvis man går for at trygdetid skal kalle hele dtoen og ha kunnskap om den kan den finne det ut selv.
Eller lage et api dedikert til det her.

Reply:
Skjønner ikke helt kommentaren din @chrissearle

Reply:
Espen og min diskusjon gitt ut på at det var ikke trygdetids ansvar å vite hvilken enum verdier i vilkaarsvurdering var yrkesskade - det var noe som hørte hjem i vilkaarsvurdering.

Men - ved å legge det på VilkaarsvurderingDto som en metode - så endres ikke json interface - men - ting som er avhengig av vilkaar model lib kan bare kalle metode på den reyhdert json etter kall.

Dvs vilkaar eier fortsatt logikken - men vi endre ikke api interface.

Det var iallefall min tolkning av Espens kommentar :)

Reply:
Enig